src/explorer/model/Dependency.ts

```diff
@@ -4,31 +4,29 @@
 import * as vscode from "vscode";
 import { getPathToExtensionRoot } from "../../utils/contextUtils";
 import { ITreeItem } from "./ITreeItem";
+import { IOmittedStatus } from "./OmittedStatus";
 import { TreeNode } from "./TreeNode";
 
-const DUPLICATE_INDICATOR: string = "omitted for duplicate";
-const CONFLICT_INDICATOR: string = "omitted for conflict";
-
 export class Dependency extends TreeNode implements ITreeItem {
     private fullArtifactName: string = ""; // groupId:artifactId:version:scope
     private _projectPomPath: string;
     private _gid: string;
     private _aid: string;
     private _version: string;
     private _scope: string;
-    private _supplement: string = "";
-    constructor(gid: string, aid: string, version: string, scope: string, supplement: string, projectPomPath: string) {
+    private _omittedStatus: IOmittedStatus;
+    constructor(gid: string, aid: string, version: string, scope: string, omittedStatus: IOmittedStatus, projectPomPath: string) {
         super();
         this._gid = gid;
         this._aid = aid;
         this._version = version;
         this._scope = scope;
         this.fullArtifactName = [gid, aid, version, scope].join(":");
-        this._supplement = supplement;
+        this._omittedStatus = omittedStatus;
```

Comment:
It's a little bit strange to use omittedStatus.status=="normal" to represent a normal dependency. 

Imagine this normal dependency has a property `omittedStatus`, and the status is "normal". So literally is it omitted?

Reply:
I think it's ok. `omittedStatus` is more like `omittedOrNot`, maybe I can rename it in a later fix.

Reply:
yes, it presents "omittedOrNot", that's why I was suggesting an optional property like `omitted?:IOmittedStatus`.

Reply:
never mind, refactor it later.